packages/web3-eth-iban/src/web3_iban.ts

```diff
@@ -0,0 +1,232 @@
+﻿/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { toChecksumAddress, isAddress, leftPad, hexToNumber, HexString } from 'web3-utils';
+import { InvalidAddressError } from 'web3-errors';
+import { IbanOptions } from './types';
+
+export class Web3Iban {
```

Comment:
```suggestion
export class Iban {
```

Make it compatible to 1.x